src/Grid.js

```diff
@@ -871,6 +871,15 @@ class Grid extends React.PureComponent {
         hoverType === 'cell'
           ? this.state.hoveredRow === rowIndex && this.state.hoveredColumn === columnIndex
           : this.state.hoveredRow === rowIndex,
+      invalid:
+        header.invalidate &&
+        header.invalidate({
```

Comment:
the name of this function should be `isValid` to avoid confusion

Reply:
i changed the name, because the function is returning message in-case of invalid data